### PR TITLE
fix ctc doc

### PR DESCRIPTION
**Pull request by AilinKid**
Signed-off-by: ailinkid <314806019@qq.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-change-column.md [hunk 1]
```diff
@@ -153,9 +153,9 @@ ERROR 8200 (HY000): Unsupported modify column: change from original type decimal
 ## MySQL 兼容性
 
 * 不支持在单个 `ALTER TABLE` 语句中进行多个更改。
-* 不支持主键列上的类型变更。
+* 不支持主键列上 Reorg-Data 类型的变更。
```
Comment (zimulala) [inline sql-statements/sql-statement-change-column.md:156]: Could we add the link for `Reorg-Data`?

Comment (AilinKid) [reply sql-statements/sql-statement-change-column.md]: sure

File: sql-statements/sql-statement-modify-column.md [hunk 2]
```diff
@@ -178,14 +178,20 @@ CREATE TABLE `t1` (
...
     ERROR 8200 (HY000): Unsupported modify column: column has primary key flag
     ```
 
+    ```sql
+    CREATE TABLE t (a int primary key);
+    ALTER TABLE t MODIFY COLUMN a bigint;
+    Query OK, 0 rows affected (0.01 sec)
+    ```
+
 * 不支持修改 generated column 的类型。例如：
```
Comment (zimulala) [inline sql-statements/sql-statement-modify-column.md:195]: s/generated column/生成列？

**Review by zimulala — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-alter-table.md [hunk 3]
```diff
@@ -147,8 +147,11 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
 
 TiDB 中的 `ALTER TABLE` 语法主要存在以下限制：
 
-* 单条 `ALTER TABLE` 语句不能完成多项操作。
-* 当前不支持有损更改，例如从 `BIGINT` 类型更改为 `INT` 类型。
+* 不支持在单个 `ALTER TABLE` 语句中进行多个更改。
+* 不支持主键列上 [Reorg-Data](/sql-statements/sql-statement-modify-column.md#Reorg-Data Change) 类型的变更。
```
Comment (TomShawn) [inline sql-statements/sql-statement-alter-table.md:151]:
```suggestion
* 不支持在主键列上进行 [Reorg-Data](/sql-statements/sql-statement-modify-column.md#Reorg-Data Change) 类型的变更。
```

File: sql-statements/sql-statement-alter-table.md [hunk 4]
```diff
@@ -147,8 +147,11 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
...
+* 不支持分区表上的列类型变更。
```
Comment (TomShawn) [inline sql-statements/sql-statement-alter-table.md:152]: 是说“不支持在分区表上进行列类型的变更”吗？

File: sql-statements/sql-statement-alter-table.md [hunk 5]
```diff
@@ -147,8 +147,11 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
...
+* 不支持生成列上的列类型变更。
+* 不支持部分数据类型（例如，部分时间类型、Bit、Set、Enum、JSON 等）的变更，因为 TiDB cast 函数与 MySQL 的行为有一些兼容性问题。
```
Comment (TomShawn) [inline sql-statements/sql-statement-alter-table.md:154]:
```suggestion
* 不支持部分数据类型（例如，部分时间类型、Bit、Set、Enum、JSON 等）的变更，因为 TiDB 中的 `CAST` 函数与 MySQL 的行为存在兼容性问题。
```

File: sql-statements/sql-statement-change-column.md [hunk 6]
```diff
@@ -153,9 +153,9 @@ ERROR 8200 (HY000): Unsupported modify column: change from original type decimal
 ## MySQL 兼容性
 
 * 不支持在单个 `ALTER TABLE` 语句中进行多个更改。
-* 不支持主键列上的类型变更。
+* 不支持主键列上 [Reorg-Data](/sql-statements/sql-statement-modify-column.md#Reorg-Data Change) 类型的变更。
 * 不支持分区表上的列类型变更。
-* 不支持 generated column 上的列类型变更。
+* 不支持生成列上的列类型变更。
 * 不支持部分数据类型（例如，部分时间类型、Bit、Set、Enum、JSON 等）的变更，因为 TiDB cast 函数与 MySQL 的行为有一些兼容性问题。
```
Comment (TomShawn) [inline sql-statements/sql-statement-change-column.md:159]:
```suggestion
* 不支持部分数据类型（例如，部分时间类型、Bit、Set、Enum、JSON 等）的变更，因为 TiDB 中 `CAST` 函数与 MySQL 的行为存在兼容性问题。
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-alter-table.md (hunk 3 above)
Comment (TomShawn) [inline sql-statements/sql-statement-alter-table.md:151]: Follow-uP: anchor error

Comment (TomShawn) [reply sql-statements/sql-statement-alter-table.md:151]: @Liuxiaozhen12 Would you please help modify the anchor in L151 of this doc and in L156 of sql-statements/sql-statement-change-column.md? Thanks!🌹

**Pull request merged**